spec/features/can_create_an_order_as_user_spec.rb

```diff
@@ -30,8 +30,20 @@
     expect(page).to have_content("20")
 
     within("#cart-donate-button") do
-      click_button "DONATE"
+      click_button "Pay with Card"
     end
+
+    sleep(2)
+    stripe_iframe = all('iframe[name=stripe_checkout_app]').last
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.